opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -175,6 +261,7 @@ message MetricDescriptor {
     // used.
     INVALID_TEMPORALITY = 0;
 
+    // TODO: Re-evaluate if this is still needed.
```

Comment:
+1 it looks like this can be dropped.

Reply:
I want to do this in a separate PR because I want to explain better why, and only that explanation is more than this entire PR most likely.

Reply:
Also I don't think it affects the overall structure, so easy to change in a separate PR.